python/ray/data/tests/test_dataset.py

```diff
@@ -3394,13 +3463,13 @@ def test_groupby_simple_multi_agg(ray_start_regular_shared, num_parts):
     random.shuffle(xs)
     df = pd.DataFrame({"A": [x % 3 for x in xs], "B": xs})
     agg_ds = ray.data.from_items(xs).repartition(num_parts).groupby(
-        lambda x: x % 3).aggregate(
-            Count(),
-            Sum(),
-            Min(),
-            Max(),
-            Mean(),
-            Std(),
+        lambda x: x % 3).apply_aggregations(
```

Comment:
This is pretty confusing. As a user, I shouldn't have to worry about two different types of aggregate-things.